webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -1606,14 +1606,36 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
         case Some(definedValueUuid) =>
           if (!apiResponse.resources.exists(_.values.values.exists(_.exists(_.valueHasUUID == definedValueUuid)))) {
             throw NotFoundException(
-              s"Value with UUID ${stringFormatter.base64EncodeUuid(definedValueUuid)} not found (maybe you do not have permission to see it, or it is marked as deleted)"
+              s"Value with UUID ${stringFormatter.base64EncodeUuid(definedValueUuid)} not found (maybe you do not have permission to see it)"
             )
           }
 
         case None => ()
       }
 
-    } yield apiResponse
+      // Check if resources are deleted, if so, replace them with DeletedResource
+      responseWithoutDeletedResources = apiResponse.resources match {
```

Comment:
Is the name of this value correct? Because deleted resources are included (but as deleted resources) in the response, arent't they?

Reply:
you're right. I renamed some of the variables to be a bit more precise